xbmc/interfaces/json-rpc/FileOperations.cpp

```diff
@@ -413,3 +414,27 @@ bool CFileOperations::FillFileItemList(const CVariant &parameterObject, CFileIte
 
   return false;
 }
+
+int CFileOperations::GetVideoDBDetails(const CVariant& parameterObject)
```

Comment:
wondering if we should move this method to JSONUtils as it shares/repeats code with `CVideoLibrary::RequiresAdditionalDetails`

Reply:
> should move this method to JSONUtils

imo, yes. it is never called directly via the api, right?

Reply:
Hence that is reason not to do so, don't want that level of dependancy in there in my opinion

Reply:
äh, yes.. disregard. looks fine like it is now. i didn't see that JSONUtils is a header only.
in any case you can do `details |= VideoDbDetailsBlah;` instead of `details = details | VideoDbDetailsBlah;` (if you like)